gqlalchemy/models.py

```diff
@@ -130,12 +131,71 @@ def __repr__(self) -> str:
         return str(self)
 
 
-class Node(UniqueGraphObject):
+class MyMeta(BaseModel.__class__):
+    def __new__(mcs, name, bases, namespace, **kwargs):  # noqa C901
+        """This creates the class `Node`. It also creates all subclasses
+        of `Node`. Whenever a class is defined as a subclass of `Node`,
+        `MyMeta.__new__` is called.
+        """
+        cls = super().__new__(mcs, name, bases, namespace, **kwargs)
+        # TODO create a discussion about accessing labels through the class definition instead of through the object. E.g. `Person.labels` instead of `person = Person("Marko"); person._node_labels`.
```

Comment:
I think we need both because accessing labels through class definition is only possible if there is a schema. Let's discuss 😄

Reply:
haha you're resolving future discussions, I love it. But yeah, I agree, both is definitely better than only one. The discussion I wanted to have is about these preceding underscores. For the class definitions we don't need the underscores because users won't set any fields/attributes to the class, but they will set attributes to the objects of the class. But then, I don't want to have different interfaces...

Reply:
Here is a cool example:

```python
class Person(Node):
    pass
    
class Student(Person):
    pass
```

In this example, the class Student will have only one label "Student", but a Student object might have multiple labels, e.g. "Person:Student". This is why we need both.